src/api/api.js

```diff
@@ -0,0 +1,24 @@
+const BASE_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=6811da81&';
```

Comment:
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=6811da81&';
```
Have you checked your app on gh-pages? You need to change the protocol to `https` to make it work, because now it's not working

https://user-images.githubusercontent.com/66570142/121379554-ce411000-c94c-11eb-8150-a95b1271b92e.mov